src/System.Private.CoreLib/shared/System/ReadOnlySpan.Fast.cs

```diff
@@ -39,7 +39,6 @@ namespace System
         /// </summary>
         /// <param name="array">The target array.</param>
         /// <remarks>Returns default when <paramref name="array"/> is null.</remarks>
```

Comment:
💡 Based on the removed text, it seems this like was originally expected to be the following. Not really necessary and the simple removal would be a fine approach.

```suggestion
         /// <remarks>Returns default when <paramref name="array"/> is <see langword="null"/>.</remarks>
```